source/wp-content/themes/wporg-showcase-2022/patterns/front-page.php

```diff
@@ -28,11 +28,11 @@
 <div class="wp-block-group alignwide" style="padding-right:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--40);padding-left:var(--wp--preset--spacing--60)"><!-- wp:group {"align":"wide","className":"wporg-hero-details","layout":{"type":"constrained"}} -->
 <div class="wp-block-group alignwide wporg-hero-details"><!-- wp:group {"align":"full","layout":{"type":"flex","orientation":"vertical","justifyContent":"right"}} -->
 <div class="wp-block-group alignfull"><!-- wp:group {"layout":{"type":"default"}} -->
-<div class="wp-block-group"><!-- wp:paragraph {"align":"left","style":{"typography":{"lineHeight":"0"}},"textColor":"white","className":"is-style-serif"} -->
-<p class="has-text-align-left is-style-serif has-white-color has-text-color" style="line-height:0"><?php esc_attr_e( 'Featuring:', 'wporg' ); ?></p>
-<!-- /wp:paragraph -->
+<div class="wp-block-group"><!-- wp:heading {"textAlign":"left","style":{"typography":{"lineHeight":"0"}},"textColor":"white","className":"is-style-serif","fontSize":"extra-large"} -->
+<h2 class="has-text-align-left is-style-serif has-white-color has-text-color has-extra-large-font-size" style="line-height:0"><?php esc_attr_e( 'Featuring:', 'wporg' ); ?></h2>
+<!-- /wp:heading -->
 
-<!-- wp:post-title {"textAlign":"left","isLink":true,"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}},"typography":{"fontStyle":"italic","fontWeight":"400"}},"textColor":"white","fontSize":"extra-large","fontFamily":"eb-garamond"} /--></div>
+<!-- wp:post-title {"level":3,"textAlign":"left","isLink":true,"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}},"typography":{"fontStyle":"italic","fontWeight":"400"}},"textColor":"white","fontSize":"extra-large","fontFamily":"eb-garamond"} /--></div>
```

Comment:
Mixed feelings about this one. Feels like a sentence to me so maybe should all be one H2

Reply:
Yeah, it's tricky because we use the `post-title`, which I think makes sense from a reusability standpoint. Maybe there's a way to target that post-title and filter it. 

On the other hand, since we do use the query block, it ends up being a list of one item with the same structure as the features list section. In that regard, it is consistent with the grid below.

Reply:
Yeah I see now. Still getting my head around building with the site editor. Definitely introduces limitations as I'm finding with the 'No search results' page.

My understanding is that a lot of screen reader users will navigate through the headings to get a sense of the page content, and I'm still a little unsure about how this will work, but I guess it will just read them consecutively, even if there is a pause. Maybe just go with it and seek a11y feedback once it's launched as you say.